pkg/admission/mutator/shoot.go

```diff
@@ -55,6 +56,28 @@ func (s *shoot) Mutate(ctx context.Context, new, old client.Object) error {
 		return fmt.Errorf("wrong object type %T", new)
 	}
 
+	// skip if shoot is in restore or migration phase
+	if wasShootRescheduledToNewSeed(shoot) {
+		return nil
+	}
```

Comment:
Q: Why do we need to check for this? Wrt to CP migration, I think this admission should only be interested in the Shoot status to check whether the lastOperation is a restore/migrate in progress.

Reply:
Ah, I now see that gardener-apiserver forbids updates in that case - https://github.com/gardener/gardener/blob/bb5f23257344259673ba5987cbec728460c3e062/plugin/pkg/shoot/validator/admission.go#L485-L487. My bad.